ZenPacks/zenoss/OpenStackInfrastructure/modeler/plugins/zenoss/OpenStackInfrastructure.py

```diff
@@ -424,7 +423,7 @@ def process(self, device, results, unused):
                 modname='ZenPacks.zenoss.OpenStackInfrastructure.AvailabilityZone',
                 data=dict(
                     id=zone_id,
-                    title=service['zone'],
+                    title=service.get('zone', ''),
```

Comment:
Also, for any titles, rather than defaulting to a blank string, default to the ID, I think that's more intuitive.